### ISSUE TITLE
Class 'AppController' not found in NiceArtisanController.php (line 13)

### ISSUE THREAD

**Issue by zden80** [bestmomo/nice-artisan#15]
Hi guys,

I try to copy installed laravel nice artisan to server and I got this error.

Class 'AppController' not found
--
in NiceArtisanController.php (line 13)

Could you pls help me?

**Comment by bestmomo** [comment]
Hi, 
Try a **composer dumpautoload** to recreate autoload.

**Comment by zden80** [comment]
but it is the problem..because I can do that on ma local machine not on server :( I don't have access there..I can just copy files to server..

**Comment by zden80** [comment]
I have copied all files and folder from localhost Vendor, Config, App. Should I copy anything else?

**Comment by bestmomo** [comment]
No, should work with all copied. Check your server configuration, PHP version...

**Comment by zden80** [comment]
I have the same version of Laravel 5.4.36 on Local machine and server. :(
I have checked the version here.. vendor\laravel\framework\src\Illuminate\Foundation\Application.php

**Comment by zden80** [comment]
what do you mean "Check your server configuration, PHP version."

**Comment by zden80** [comment]
Hello! Any news for this ?  :(

**Comment by bestmomo** [comment]
Hello, what is the PHP version ?

**Comment by zden80** [comment]
7.1

**Comment by zden80** [comment]
How or where should I put this in composer.json file?

this is my composer file.

{
  "name": "bestmomo/nice-artisan",
  "description": "Web interface for Laravel Artisan",
  "homepage": "http://github.com/bestmomo/nice-artisan",
  "license": "MIT",
  "authors": [
    {
      "name": "Bestmomo",
      "email": "grandheretique@free.fr",
      "homepage":"http://laravel.sillo.org"
    }
  ],
  "require": {
    "php": ">=5.6.4",
    "illuminate/support": "~5.4",
    "bestmomo/nice-artisan": "1.0.*"
  },
  "autoload": {
    "psr-4": {
      "Bestmomo\\NiceArtisan\\": "src/"
    }
  }
}

**Comment by bestmomo** [comment]
Sorry but I have no idea for your issue... Composer file is ok.

**Comment by marcelohcortez** [comment]
If anyone still has this issue. One possible solution is to update your 'app.php' inside your 'www/config' folder.
I had to do that because I extended the original plugin, since it's breaking in a lot of places.
In the 'aliases' block, you must add:
'AppController' => App\Http\Controllers\Controller::class
